3-Write-your-own-network_en.ipynb

```diff
@@ -415,7 +408,8 @@
     "        h = F.relu(self.bn(self.conv(x)))\n",
     "        if self.pool_drop:\n",
     "            h = F.max_pooling_2d(h, 2, 2)\n",
-    "            h = F.dropout(h, ratio=0.25, train=self.train)\n",
+    "            chainer.using_config('train', self.train)\n",
```

Comment:
I think this line is not needed. `F.dropout` automatically switch the mode according to the global configuration, and it should be able to be the test mode when the global config of `train` is set to `False` to use `Trainer` and `Evaluator`.